its/plugin/tests/src/test/java/com/sonar/it/java/suite/PackageInfoTest.java

```diff
@@ -74,7 +74,7 @@ public void should_take_into_account_package_annotations() {
     orchestrator.executeBuild(build);
 
     List<Issue> issues = TestUtils.issuesForComponent(orchestrator, projectKey);
-    assertThat(issues).hasSize(11);
+    assertThat(issues).isEmpty();
```

Comment:
We should understand why?

Reply:
This test use a profile `profile-package-info-annotations.xml` that activate only symbolic execution ruiles (S2259, S2589). This is why the test raise no issue without the SE rules. We need to remove the test.